server/lib/exporters/__init__.py

```diff
@@ -120,23 +120,21 @@ def dump_and_checksum(self, func, zip_path):
         for alg in self.algs:
             self.state[alg].append((zip_path, getattr(hash_file_stream, alg)))
 
+    def _agg_index_by_uri(self, uri):
+        aggs = self.manifest_obj.manifest["aggregates"]
+        return next((i for (i, d) in enumerate(aggs) if d['uri'] == uri), None)
+
     def append_aggergate_checksums(self):
         """
         Takes the md5 checksums and adds them to the files in the 'aggregates' section
         :return: None
         """
+        aggs = self.manifest_obj.manifest["aggregates"]
         for path, chksum in self.state['md5']:
-            uri = '../' + path
-            index = next(
-                (
-                    i
-                    for (i, d) in enumerate(self.manifest['aggregates'])
-                    if d['uri'] == uri
-                ),
-                None,
-            )
+            uri = './' + path
```

Comment:
```suggestion
            uri = "./" + path.replace("data/", "", 1)
```
The md5 values currently aren't included I think because of the `data/` path prefix.